docs/wallet/universal-links.md

```diff
@@ -269,19 +269,32 @@ const finishUrl = encodeURIComponent("https://my-app.org/finish");
 // Base64 encode the verification request
 const base64EncodedRequest = btoa(JSON.stringify(request));
 
-// URL encode the verification request with the parameters
-const encodedRequestWithParams =(`i_m=${base64EncodedRequest}&back_url=${backUrl}&finish_url=${finishUrl}`);
 
-// Open the Web Wallet URL with the encoded request
-window.open(`https://wallet-dev.privado.id/#${encodedRequestWithParams}`);
+// Configure the Wallet URL(universal link)
+walletUrlWithMessage = `https://wallet-dev.privado.id/#i_m=${base64EncodedRequest}&back_url=${backUrl}&finish_url=${finishUrl}`;
 
-//Note:
+// Open the Wallet URL to start the verification process
+window.open(walletUrlWithMessage);
+
+
+/*
+Note
+=================================================
+
+// You can also use the `request_uri` parameter instead of `i_m`. 
+// For that, first define the URL containing the request, and URI encode it.
 
-// You can also use the `request_uri` param instead of `i_m`, For that first define the Url containing the request.
 const requestUrl = encodeURIComponent("https://raw.githubusercontent.com/0xpulkit/Examples_Privado-ID/main/KYCV3.json");
 
-// Configure the Web Wallet URL
+// Configure the Wallet URL using `request_uri` instead of `i_m`
 walletUrlWithRequestUri = `https://wallet-dev.privado.id/#request_uri=${requestUrl}&back_url=${backUrl}&finish_url=${finishUrl}`);
```

Comment:
Same. We should always point to production.

Reply:
Fixed it